### ISSUE TITLE
Slow startup

### ISSUE THREAD

**Issue by alexchandel** [AtomLinter/linter-rust#49]
`linter-rust` adds 28ms to my startup time, compared to `37ms` for `linter` and `0ms` for everything else. This startup time should be <= 5ms.

**Comment by Maplicant** [comment]
I don't think that the team should focus on improving the startup time by 23ms. Blinking takes 300-400ms, that's how little it matters.
